tests/ui/needless_raw_string.rs

```diff
@@ -9,8 +9,7 @@ fn main() {
     br#"aaa"#;
```

Comment:
All these tests seem to be single line examples. Could you add a test where the string spans multiple lines?

Reply:
👍 added